### DISCUSSION TITLE
how to search for available versions of a thing?

### DISCUSSION THREAD

**Discussion by airtonix** [aquaproj/aqua#2600] [Deprecated (Question and Answer)] [answered]
### Question

currently search only provides the lates version of a thing.

but it's obviously useful to know what other versions are availanle.

**Comment by suzuki-shunsuke** [comment]
aqua supports getting versions from GitHub Releases and GitHub tags.

**Reply by suzuki-shunsuke** [reply, marked as answer]
If you want to select versions, `aqua g`'s `-s` option is useful.

```sh
aqua g -s cli/cli
```
